cmd/panic/internal/utf8/utf8.go

```diff
@@ -3,6 +3,8 @@
 // that can be found in the LICENSE file.
 
 // Package ùtf8 tests a package and function with non-ASCII names.
+// The filename itself does not contain unicode as this causes issues
```

Comment:
Please insert an empty line "//" so this is in a separate paragraph. It makes godoc output much nicer.

Reply:
Done.